draft-ietf-rats-architecture.md

```diff
@@ -1317,3 +1317,160 @@ use beyond the period for which it deems the Attestation Result to remain
+
+Relying Party and Relying Party Owner:
+
+: are also the application vendor cloud service; since the Verify and Relying Part are the same the distinction between passport and background check doesn’t apply.
+
+Attestation Result:
+
+: non existent as the Verifier is the same as the Rely Party
+
+## System Example Base on a Third Party Verification Service
+
+Three entities (companies / organizations) participate in this example:
+
+First off, there is a device maker. It does the following:
+* Manufacturers devices using HW and SW of their choice that includes an Attester
+* Puts an attestation key into each device (symmetric or asymmetric)
+* Has the device certified by one or more certification programs
+* Generates a detailed Endorsement for each device that includes the verification key, list of certifications, reference measurement values and implicit claims
+
+The Relying Party operates some cloud service for which attestation needed, perhaps a payment service. It does the following:
+* Receives service requests from devices. The service requests includes Attestation Evidence.
+* Sends the Attestation Evidence to a 3rd party verification service.
+* Receives Attestation Results from 3rd party verification service.
+* Decides whether to grant the service, perhaps varying the payment amount allowed based on details that are in the Attestation Result such the certifications the device has received
+
+The 3rd Party Verification and Attestation Aggregation Service. It does the following:
+* Receives Attestation Evidence from the Relying Party (This uses the Background Check Model)
+* Looks up the Endorsement by an Endorsement identifier, perhaps a URI, and receives the Endorsement from the device maker, most likely over a channel that has both confidentiality, integrity protection and authentication.
+* Performs the Verification of the Attestation Evidence which involves: verifying the signature on the Attestation Evidence and comparison of claims to reference values in the Endorsement.
+* Generates the Attestation Results which involves making some new values that report the result of comparison to the reference values and copying implied claims from the Endorsements and copying some claims from the Attestation Evidence
+
+This maps as follows in the RATS architecture:
+
+Endorser:
+
+: The device maker
+
+Endorsement:
+
+: A detailed message created by the device maker that is secured for confidentiality that contains reference values, the verification key and implicit claims
+
+Attester: 
+
+: The HW and SW made by the device maker
+
+Attestation Evidence:
+
+: A protocol message generated by the Attester sent to the Relying Party
+
+Verifier and Verifier Owner:
+
+: A third-party Attestation service that can process Attestation Evidence from many different device makers and provide results to many different Relying Parties.
+
+Relying Party and Relying Party Owner:
+
+: The operator of a cloud-based payment service.
```

Comment:
No, the Relying Party is the payment service itself.  The operator of it is the Relying Party Owner.